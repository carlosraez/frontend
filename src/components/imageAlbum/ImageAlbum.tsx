import { Photo } from '../../types/photos'
import './ImageAlbun.styles.css';
interface Props {
  photosAlbun: Array<Photo>
}

export const ImageAlbum = ({photosAlbun}:Props) => 
   (<>
    {photosAlbun?.map((photo) => (
        <img key={photo.id} src={photo.previewUrl} alt={photo.name} />
      ))}
  </>)

