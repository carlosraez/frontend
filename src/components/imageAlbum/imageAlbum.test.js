import { render } from '@testing-library/react'
import { ImageAlbum } from './ImageAlbum.tsx'


describe('MainScreen tests', () => { 
    test('should getDataPhotoAlbun should be return a list off images',  () => {
       const photos = [ 
        {
         id: '123445',
         name: 'foto1', 
         previewUrl:'https://media.gettyimages.com/photos/two-medicine-lake-sunrise-picture-id827422252?s=612x612'
        },
        {
          id: '123446',
          name: 'foto2', 
          previewUrl:'https://media.gettyimages.com/photos/two-medicine-lake-sunrise-picture-id827422252?s=612x612',
        }] 
       const { containerÂ } = render(<ImageAlbum photosAlbun={photos} />)
       expect( container ).toMatchSnapshot()  
     })
 })