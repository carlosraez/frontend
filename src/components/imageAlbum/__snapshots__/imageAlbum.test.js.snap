// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainScreen tests should getDataPhotoAlbun should be return a list off images 1`] = `
<div>
  <img
    alt="foto1"
    src="https://media.gettyimages.com/photos/two-medicine-lake-sunrise-picture-id827422252?s=612x612"
  />
  <img
    alt="foto2"
    src="https://media.gettyimages.com/photos/two-medicine-lake-sunrise-picture-id827422252?s=612x612"
  />
</div>
`;
